python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Commands.xml

```diff
@@ -24,6 +24,18 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_command_changeAlignment</EFDB_Topic>
+    <item>
+      <EFDB_Name>optical_configuration</EFDB_Name>
```

Comment:
topic attributes must be `lowerCamelCase` (yes, I know it is confusing, sorry). Also, I would rename this to something shorter and maybe more expressive? maybe just `configuration`?